tests/fixtures/data.yml

```diff
@@ -9,12 +9,16 @@ person:
     ORIGINAL_HIRE_DATE: !!timestamp 2001-01-01 00:00:00.0
     APPOINTMENT_END_DATE: !!timestamp 2010-01-01 00:00:00.0
     JOB_TITLE: Adjunct Professor
-    PERSONNEL_SUBAREA_CODE: CFAC
-    HR_ORG_UNIT_TITLE: Chemisty
+    PERSONNEL_SUBAREA_CODE: CFAT
+    HR_ORG_UNIT_ID: "1"
+    HR_ORG_LEVEL4_NAME: Chemistry
```

Comment:
This is a lie. The actual query uses the value from `dlc` but the cli tests don't seem to spin these up and do a real query? Not sure. This was a hack to get the build working again. It's probably better to address this now, but I wanted some eyes on this before I went further.